depmap_omics_upload/tracker.py

```diff
@@ -138,13 +129,11 @@ def insert_to_seq_table(self, df):
             self.seq_table_name, df, convert_to_custom_names=False, inplace=True
         )
         self.client.insert_only(self.seq_table_name, df)
-        self.client.commit()
```

Comment:
Is `commit` also redundant in the new client?

Reply:
Correct